actionview/lib/action_view/helpers/date_helper.rb

```diff
@@ -90,12 +90,17 @@ module DateHelper
       #   distance_of_time_in_words(from_time, from_time + 50.minutes, scope: 'datetime.distance_in_words.short') # => "an hour"
       #   distance_of_time_in_words(from_time, from_time + 3.hours, scope: 'datetime.distance_in_words.short')    # => "3 hours"
       def distance_of_time_in_words(from_time, to_time = 0, options = {})
+        if [from_time, to_time].any?(&:nil?)
```

Comment:
I think that

```
if from_time.nil? || to_time.nil?
```

would be quicker, allocate no new objects and is no less readable.